.circleci/main.yml

```diff
@@ -1163,7 +1163,60 @@ jobs:
                 echo "regular release."
                 gh release create --draft ${CIRCLE_TAG} --title ${CIRCLE_TAG:1} ./dist-pach/pachctl/pachctl_${CIRCLE_TAG:1}* ./dist-pach/mount-server/mount-server_${CIRCLE_TAG:1}* --generate-notes
             fi
+  console-release-draft:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - gh/setup:
+          version: 2.13.0
+      - run:
+          name: create github console release draft
+          command: |
+            if [[ $CIRCLE_TAG == *"-"* ]];
+              then
+                  echo "git tag has - assuming its a prerelease."
+                  parentBranch=master
+              else
+                  echo "regular release."
+                  parentBranch=${CIRCLE_TAG:1}
+              fi
+            git clone https://github.com/pachyderm/haberdashery.git haberdashery
+            git checkout -b "release-${CIRCLE_TAG:1}"
+            tmp=$(mktemp)
+            jq --arg val "${CIRCLE_TAG:1}-${CIRCLE_SHA1}" '.pachyderm = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_SHA1}" '.pachReleaseCommit = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_TAG:1}" '.pachctl = $val' version.json > "$tmp" && mv "$tmp" version.json
+            git commit -m "bump pachyderm version"
+            git push
+            PR=$(gh pr create --base $parentBranch --title "bump pach version for release ${CIRCLE_TAG:1}" --repo pachyderm/haberdashery)
+            gh pr merge ${PR} --auto --repo pachyderm/haberdashery
+            gh pr checks ${PR} --10 --watch --repo pachyderm/haberdashery
+            sleep 2 # in case we create release to fast
+            if [[ $CIRCLE_TAG == *"-"* ]];
+              then
+                  echo "git tag has - assuming prerelease."
+                  gh release create --draft ${CIRCLE_TAG:1}-1 --tittle ${CIRCLE_TAG:1}-1 --target $parentBranch --generate-notes --prerelease --repo pachyderm/haberdashery
+              else
+                  echo "regular release."
+                  gh release create --draft ${CIRCLE_TAG:1}-1 --tittle ${CIRCLE_TAG:1}-1 --target $parentBranch --generate-notes --repo pachyderm/haberdashery
+              fi
+            SECONDS=0
+            while :; do
+                docker pull pachyderm/haberdashery:${CIRCLE_TAG:1}-1 2>/dev/null && break
+                if (( $SECONDS > 1 )); then
```

Comment:
how does `SECONDS` get incremented here?

Reply:
`SECONDS` is a special variable in bash whose value is incremented once per second.

Reply:
I had no idea! I wonder if this has broken any scripts I've ever written thinking SECONDS was just a normal variable.... that's horrifying